dev/devicelab/manifest.yaml

```diff
@@ -509,6 +509,15 @@ tasks:
     required_agent_capabilities: ["mac/ios"]
     timeout_in_minutes: 20
 
+  flutter_driver_screenshot_test_ios:
+    description: >
+      Screenshot tests running on iPhone 6s.
+    stage: devicelab_ios
+    required_agent_capabilities: ["mac/ios, ios/gl-render-image"]
```

Comment:
Added a new capability to make the test only run on one specific ios device

Reply:
Change to `["mac/ios", "ios/gl-render-image"]`, (make sure to quote each term).

Make the description be more specific, explain it makes sure flutter renders an fixed image on ios GL.

Reply:
Thanks! I updated the PR.